x-pack/platform/plugins/shared/alerting/server/routes/rule/apis/create/create_rule.examples.yaml

```diff
@@ -0,0 +1,31 @@
+# Because of where this file is linked to the code you are in
```

Comment:
Q: Is there a way to create the template of these files via a command?

I worry that documenting examples might be too complex that ends up not being used. WDYT?

Reply:
Yeah, that's a good question. I'll expand our existing tutorial to guide devs through this. Perhaps some tooling could come later!